sources/packages/web/src/views/student/StudentApplicationDetails.vue

```diff
@@ -0,0 +1,169 @@
+<template
+  ><div class="p-m-4">
+    <h5 class="text-muted">
+      <a @click="goBack()">
+        <v-icon left> mdi-arrow-left </v-icon> Back to Applications</a
+      >
+    </h5>
+    <h1><strong>Financial Aid Application</strong></h1>
+    <v-btn color="primary" class="float-right ml-2" @click="toggle"
+      >Application Options
+      <v-icon size="25"> mdi-arrow-down-bold-circle</v-icon></v-btn
+    >
+    <Menu class="mt-n15" ref="menu" :model="items" :popup="true" />
+    <v-btn
+      color="primary"
+      class="p-button-raised ml-2 float-right"
+      @click="
+        $router.push({
+          name: StudentRoutesConst.ASSESSMENT,
+          params: {
+            applicationId: id,
+          },
+        })
+      "
+    >
+      <v-icon size="25">mdi-text-box-plus</v-icon>
+      View Assessment
+    </v-btn>
+    <CancelApplication
+      :showModal="showModal"
+      :applicationId="id"
+      @showHideCancelApplication="showHideCancelApplication"
+      @reloadData="getApplicationDetails"
+    />
+
+    <v-container class="pt-12">
+      <div
+        class="bg-white application-info-border"
+        v-if="
+          applicationDetails.applicationStatus == ApplicationStatus.cancelled
```

Comment:
== or === ?

Reply:
mm, ya we need to check the type too. updating